addons/medical_treatment/stringtable.xml

```diff
@@ -3442,12 +3456,15 @@
         </Key>
         <Key ID="STR_ACE_Medical_treatment_convertItems_DisplayName">
             <English>Convert vanilla items</English>
+            <German>Standard Arma-Items in ACE-Items umwandeln</German>
         </Key>
         <Key ID="STR_ACE_Medical_treatment_convertItems_Description">
             <English>Enables or disables whether vanilla medical items are converted to ACE medical items or just removed</English>
+            <German>Sollen Standard Medic-Items in ACE-Items umgewandelt oder entfernt werden?</German>
```

Comment:
English version is not a question.

Reply:
What about "Wandel Medic-Items in ACE-Items umgewandelt um, ansonsten entferne diese." ?

Reply:
I'd say just translate the english more directly.
"Aktiviert, ob Standard Medic-Items in ACE Varianten umgewandelt, oder entfernt werden"
Not sure about the comma there, but it makes it easier to read.
Aktiviert doesn't sound that good as you have two choices.
Maybe
"Entscheidet, ob..."

Also "Medic-Items" is that used consistently in other translations?

`STR_ACE_Medical_treatment_convertItems_DisplayName` says "Arma-Items" instead of "Standard Medic-Items"
I think "Standard Medic-Items" sounds better.
`STR_ACE_Medical_treatment_convertItems_DisplayName` should be changed to that